src/lib/Server/Output/ValueObjectVisitor/ImageVariation.php

```diff
@@ -59,5 +59,7 @@ protected function visitImageVariationAttributes(Visitor $visitor, Generator $ge
             $generator->startValueElement('fileSize', $data->fileSize);
             $generator->endValueElement('fileSize');
         }
+
+        $visitor->setHeader('Content-Type', $generator->getMediaType('ContentImageVariation'));
```

Comment:
Correct me if I'm wrong, but shouldn't this go to the `visit` method, as it has no connection with going over the attributes?